CMakeLists.txt

```diff
@@ -180,30 +215,33 @@ target_link_libraries(unassemblize PRIVATE
     readerwriterqueue
 )
 
-target_include_directories(unassemblize PRIVATE
-    .
-    src
-    3rdparty/span/include
-)
-
 if(WINDOWS)
-    target_sources(unassemblize PRIVATE
-        wincompat/strings.h
-    )
-
     target_link_libraries(unassemblize PRIVATE
-        d3d9.lib # How does it find it???
-        imgui_win32
```

Comment:
Where is this linked now?

Reply:
Mistakenly removed - it's now been restored